docs/user-guide/pod-security-policy/index.md

```diff
@@ -137,12 +137,12 @@ A pod must validate every field against the PSP.
 Here is an example Pod Security Policy. It has permissive settings for
 all fields
 
-{% include code.html language="yaml" file="psp.yaml" ghlink="/docs/user-guide/pod-security-policy/psp.yaml" %}
+{% include code.html language="json" file="psp.json" ghlink="/docs/user-guide/pod-security-policy/psp.json" %}
```

Comment:
https://github.com/kubernetes/kubernetes.github.io/pull/3010 is fixing this to use yaml like it should be.  Recommend closing this and taking that update.